docs/modules/ROOT/pages/erc20.adoc

```diff
@@ -120,8 +121,7 @@ mod MyToken {
 }
 ----
 
-In order for the `MyToken` contract to integrate the ERC20 component, it instantiates both the `ERC20Impl` and `ERC20MetadataImpl`.
-The `ERC20Impl` instances embeds the implementation and exposes the `IERC20` interface.
+`MyToken` instantiates both the `ERC20Impl` and `ERC20MetadataImpl` with the embed directive which integrates the implementations into the contract.
```

Comment:
```suggestion
`MyToken` integrates both the `ERC20Impl` and `ERC20MetadataImpl` with the embed directive which marks the implementations as external in the contract.
```
i'm unsure about using this word, it's not actually an instance isn't it? also wanted to make sure we say they're exposed (integrating the impl could be without exposing it)

Reply:
> i'm unsure about using this word, it's not actually an instance isn't it?

I suppose it depends on how we define instance. IMO a component impl with a generic state that gets actualized with a specific contract state becomes an instantiation. No issue with using "integrates" though

> also wanted to make sure we say they're exposed (integrating the impl could be without exposing it)

Yes, good call